### PR TITLE
Balanced accuracy

### PR DESCRIPTION
**Pull request by eighthill**
Hey guys, I've added the code to calculate the balanced accuracy.
Small overview:

reset()
Sets all variables to torch.zeros. This method is called everytime when the class is being initialized.

update()
Converts logits to the predicted class and updates counts for correctly classified and total per class and assign them to a variable respectively.

getBACC()
Calculates balanced accuracy by determining the accuracy for each individual class, then computes the mean for the class accuracies and returns the output as python scalar.

### CODE REVIEW

**Comment by SophZoe** [comment]
Code looks good, i can't find any noticeable errors. Ready to be merged.

**Review by SophZoe — COMMENTED** [review]
.

**Review by SophZoe — APPROVED** [review]
Approved. (:
